distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/cli/PluginSecurity.java

```diff
@@ -34,6 +34,9 @@
  */
 public class PluginSecurity {
 
+    public static final String ENTITLEMENTS_DESCRIPTION_URL =
```

Comment:
This is duplicated in https://github.com/elastic/elasticsearch/pull/125207, but should be easy to merge later